src/evict/evict_lru.c

```diff
@@ -968,7 +1003,8 @@ __wt_evict_file_exclusive_on(WT_SESSION_IMPL *session)
      * any existing LRU eviction walk for the file.
      */
     (void)__wt_atomic_addv32(&cache->pass_intr, 1);
-    WT_WITH_PASS_LOCK(session, ret = __evict_clear_walk_and_saved_tree_if_current_locked(session));
+    WT_WITH_PASS_LOCK(
+      session, ret = __evict_clear_walk_and_saved_tree_if_current_locked(session, false));
```

Comment:
I see this matches the clear_pos boolean logic added in the other function. Can you elaborate on why you're doing it this way?